Source/Urho3D/Graphics/DebugRenderer.cpp

```diff
@@ -505,7 +541,7 @@ void DebugRenderer::Render()
     graphics->SetBlendMode(lineAntiAlias_ ? BLEND_ALPHA : BLEND_REPLACE);
     graphics->SetColorWrite(true);
     graphics->SetCullMode(CULL_NONE);
-    graphics->SetDepthWrite(true);

```

Comment:
Since this is DebugRenderer-wise change, what noticeable changes in rendering of debug geometry could appear? For example..
If depth write is disabled, degug geometry will be rendered in draw order without Z-test with another debug geometry. Am I right? Is it ok?